payments/swish/after-payment.md

```diff
@@ -107,7 +107,8 @@ request for the given operation.
 
 ## Swish transactions
 
-All Swish transactions are described below.
+All Swish transactions are described below. Please note that Swish does not
+support Merchant Initiated Transactions for the time being.
```

Comment:
Do we have somewhere MIT is described, or is that not done yet? If we do, can we link to it?

Reply:
MIT is described in an include added in card payments other features. I'm on my way to add some changes on it right now, so I'll add a link now! I thought you had given it the thumbs up, but I see that only Marina did it before I merged.